packages/common/src/typing-indicator/typing-indicator.tsx

```diff
@@ -1,16 +1,15 @@
-import React, { FC, useState, useEffect, useCallback, useRef } from "react";
+import { useState, useEffect, useCallback, useRef } from "react";
 import { usePubNub } from "pubnub-react";
 import { useAtom } from "jotai";
+import isEqual from "lodash.isequal";
```

Comment:
I still don't get why we install a lib by lib instead of tree-shaken import isEqual from lodash/isEqual :P